docs/features/map-tiles-custom-areas/high-level-design.md

```diff
@@ -1,43 +1,76 @@
 # Map tiles for custom planning areas and custom grids - High-level design
 
-This document aims at providing an overview of the architecture of the tiles to fulfill the [requirements document](./requirements.md) for the Marxan platform.
+This document aims at providing an overview of the architecture of the tiles to
+fulfill the [requirements document](./requirements.md) for the Marxan platform.
 
-The tiles needed to render all the kinds of areas reviewed in this document will be of two kinds:
+The tiles needed to render all the kinds of areas reviewed in this document will
+be of two kinds:
 
-preview: these are used when a user is configuring planning area and grid for a project, before the project is actually created in the system (hence, we won't have a project_id available at this stage, nor will any spatial data have been linked to a project yet, even though it will have been stored in the geodb
-for scenario: these are used when project-level planning area and grid data is shown while creating or editing a scenario; at this stage the relevant spatial data will have been linked to the scenario's parent project
+preview: these are used when a user is configuring planning area and grid for a
+project, before the project is actually created in the system (hence, we won't
+have a project_id available at this stage, nor will any spatial data have been
+linked to a project yet, even though it will have been stored in the geodb for
+scenario: these are used when project-level planning area and grid data is shown
+while creating or editing a scenario; at this stage the relevant spatial data
+will have been linked to the scenario's parent project
```

Comment:
I didn't notice this in the HLD PR - is just a final full stop missing, or are we missing part of the sentence?